src/scripts/main.coffee

```diff
@@ -1023,6 +1030,8 @@ class Formbuilder
       DEFAULT_TIME: 'options.default_time'
       DEFAULT_DATE: 'options.default_date'
       REFERENCE_ID: 'reference_id'
+      INLINE_IMAGES_ENABLED: 'options.inline_images_enabled',
```

Comment:
Pretty sure it's harmless but you shouldn't need the comma on the end of this line